### ISSUE TITLE
Lenovo Tab 4 10 Plus

### ISSUE THREAD

**Issue by Romank1** [rovo89/Xposed#270]
Lenovo Tab 4 10 Plus TB-X704L
Android N 7.1.1
CPU arm64

I try to install Xposed Framework 88.1 via  XposedInstaller v3.1.2 and TWRP, but the installation stucks.

![image](https://user-images.githubusercontent.com/4432037/31816494-3f7693ba-b599-11e7-93b0-2ff71d5cea63.png)

Can you check this please.
I can test the debug versions, if you want.

Roman

**Comment by rovo89** [comment]
I can't reproduce this... do you have similar issues for other ZIPs?

**Comment by Romank1** [comment]
> I can't reproduce this... do you have similar issues for other ZIPs?

No, I successfully installed Magisk and Systemless Xposed v88.1 via TWRP.
